README.md

```diff
@@ -1,84 +1,9 @@
 # KubeStellar Helm Repository
 
-Hello world!  I'm a Helm repository for KubeStellar charts.  We currently support v0.23.0 and newer versions of our KubeStellar Core components.
+Hello world!  I'm a potential Helm repository for KubeStellar charts.
 
-## Pre-Reqs
-See [Pre-Reqs](https://docs.kubestellar.io/release-0.23.0/direct/pre-reqs/)
+Visit [the KubeStellar website](https://kubestellar.io) to learn about KubeStellar.
 
-The information provided is specific for the following release:
+To see a simple example of setting up and using KubeStellar, see [its quickstart](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). That includes using its "Core" Helm chart. There is also [full documentation of that chart](https://docs.kubestellar.io/release-0.23.1/direct/core-chart/).
```

Comment:
suggest using latest alias instead of a specific release number, i.e.
https://docs.kubestellar.io/latest/direct/get-started/      
and 
https://docs.kubestellar.io/latest/direct/core-chart/
That way this intro only needs to be rewritten if/when one of those files goes away.

Reply:
done